src/CommunityToolkit.Maui/Behaviors/BaseBehavior.shared.cs

```diff
@@ -93,29 +52,17 @@ protected override void OnDetachingFrom(TView bindable)
 	/// <param name="property"></param>
 	/// <param name="defaultBinding"></param>
 	/// <returns></returns>
-	[MemberNotNullWhen(true, nameof(bindingField), nameof(getContextMethod))]
+	[Obsolete($"{nameof(IsBound)} is no longer used by {nameof(CommunityToolkit)}.{nameof(CommunityToolkit.Maui)} and will be removed in a future release")]
```

Comment:
I was able to remove the need for this method which removes the need for Reflection.

However, I've kept it here and marked it `[Obsolete]` because removing it would be a breaking change to the API. We can remove this method for the next major version bump.